plugin-build/build.gradle.kts

```diff
@@ -70,3 +70,40 @@ if (gradle.gradleVersion >= "6.6.0") {
     val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
     publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
 }
+
+apply<DistributionPlugin>()
+
+val sep = File.separator
+
+configure<DistributionContainer> {
+    this.getByName("main").contents {
+        from("build${sep}libs")
+        from("build${sep}publications${sep}maven")
+        from("build${sep}publications${sep}sentryPluginPluginMarkerMaven")
+    }
+}
+
+tasks.named("distZip").configure {
+    this.dependsOn("publishToMavenLocal")
+    this.doLast {
+        val distributionFilePath = "${this.project.buildDir}${sep}distributions" +
+            "${sep}${this.project.name}-${this.project.version}.zip"
+        val file = File(distributionFilePath)
+        if (!file.exists()) {
+            throw IllegalStateException(
+                "Distribution file: $distributionFilePath does not exist"
+            )
+        }
+        if (file.length() == 0L) {
+            throw IllegalStateException("Distribution file: $distributionFilePath is empty")
+        }
+    }
+}
```

Comment:
Can you provide some context on why this is needed (like who is supposed to consume this)?

Reply:
Updated the PR description.